### DISCUSSION TITLE
Dynamic options for form values?

### DISCUSSION THREAD

**Discussion by zgutterm** [community/community#29067] [Projects and Issues]
Is it possible to dynamically generate options in an issue form template? For example, I want to ask the person reporting the issue which version of the software the issue occurred in. I don't want to manually update the template every time there is a new version released (and different repos are using different versioning). Is there way to pull in, say, the tags or something as an option for versions?

**Comment by jamacku** [comment]
@zgutterm I'm afraid that this isn't currently supported.

Idea: you can set up a workflow to update issue form templates on tag/release events.

**Comment by ShaMan123** [comment]
#4299 

I needed that as well so I ended up writing an [action](https://github.com/ShaMan123/gha-populate-form-version) for it
